pip/req/req_install.py

```diff
@@ -48,7 +48,7 @@ class InstallRequirement(object):
 
     def __init__(self, req, comes_from, source_dir=None, editable=False,
                  link=None, as_egg=False, update=True, editable_options=None,
-                 pycompile=True, markers=None, isolated=False):
+                 pycompile=True, markers=None, isolated=False, options=None):
```

Comment:
ah... I see.  yea, I honestly wasn't thinking about that.   ok in this case, but  I'm happy either way.  I'd almost expect pylint/pep8 to have added a check for mutable kwargs?